etc/telegraf_windows.conf

```diff
@@ -105,10 +105,11 @@
       "% Privileged Time",
       "% User Time",
       "% Processor Time",
+      "% DPC Time",
     ]
     Measurement = "win_cpu"
     # Set to true to include _Total instance when querying for all (*).
-    #IncludeTotal=false
+    IncludeTotal=true
```

Comment:
To match the default of Unix cpu plugin, which has totalcpu=true: [telegraf.conf](https://github.com/influxdata/telegraf/blob/20bf90ee52d7be1d2f0864b05b0d75ed2da2f5c2/etc/telegraf.conf#L490)